launch_workers_qsub.sh

```diff
@@ -20,7 +15,7 @@
 echo JOB_ID: $PBS_JOBID JOB_NAME: $PBS_JOBNAME HOSTNAME: $PBS_O_HOST
 echo start_time: `date`
 
-## LOAD PYTHON VENV:
+## Load python virtual environment
 . /ihome/cwilmer/kbs37/venv/ipmof/bin/activate
 
 cd $PBS_O_WORKDIR
```

Comment:
we're going to want this as a .sample script, with your production script gitignored, like we do with the other samples. Otherwise you and Yan will be fighting over which virtualenv to use.